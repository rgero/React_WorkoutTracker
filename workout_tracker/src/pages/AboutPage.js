import Container from 'react-bootstrap/Container';

function AboutPage() {
    return (
        <Container>
            <a href="https://www.flaticon.com/free-icons/gym" title="gym icons">Gym icons created by Freepik - Flaticon</a>
        </Container>
    );
  }
  
  export default AboutPage;