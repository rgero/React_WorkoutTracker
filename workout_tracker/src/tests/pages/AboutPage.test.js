import renderer from 'react-test-renderer';
import AboutPage from '../../pages/AboutPage';

test('Renders About Page and checks against snapshot', ()=> {
    const pageRender = renderer.create(<AboutPage />).toJSON();
    expect(pageRender).toMatchSnapshot();
})