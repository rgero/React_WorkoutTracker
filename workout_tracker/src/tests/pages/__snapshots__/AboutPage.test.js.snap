// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders About Page and checks against snapshot 1`] = `
<div
  className="container"
>
  <a
    href="https://www.flaticon.com/free-icons/gym"
    title="gym icons"
  >
    Gym icons created by Freepik - Flaticon
  </a>
</div>
`;
